tutor/resources/styles/global/navbar.less

```diff
@@ -217,7 +217,6 @@
       border: 0;
       background-color: transparent;
       width: 100%;
-      text-align: left;
```

Comment:
because that rule was only there to override the global `justified` one.  Now that that's  removed, it isn't needed any longer.  I did a search for `text-align` `left` and that was the only one I found.  There's probably others but they weren't found by my simple grep